db/schema.rb

```diff
@@ -54,8 +54,8 @@
     t.datetime "updated_at",                          null: false
   end
 
-    t.index ["email"], name: "index_users_on_email", unique: true, using: :btree
-    t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true, using: :btree
+  add_index "users", ["email"], name: "index_users_on_email", unique: true, using: :btree
+  add_index "users", ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true, using: :btree
   
 
 end
```

Comment:
To be clear, you should **never**  be directly editing the schema.rb file.  It is a dump for all the changes you should be making through valid migrations.  If you have been doing this, please revert all changes and do it through migrations or we will have no valid migration tree to undo anything you've done or change things.